scripts/esm/camera-frame.mjs

```diff
@@ -108,45 +106,54 @@ class Ssao {
      */
     type = SsaoType.NONE;
 
+    /**
+     * @visibleif {type !== 'none'}
```

Comment:
@mvaligursky This is another reason why I wanted an `enabled` property on SSAO. It's gonna look quite incongruous against the other effects in the Editor UI (where everything else can be toggle in the effect header bar.

Reply:
could we have a `type=combine|lighting` and a separate enabled field?

Reply:
sure, feel free to change this script